man/faq/external-vector.Rmd

```diff
@@ -0,0 +1,68 @@
+
+```{r, child = "setup.Rmd", include = FALSE}
+```
+
+## Ambiguity between columns and external variables
+
+In selecting functions like `dplyr::select()` or `tidyr::pivot_longer()`,
+you can refer to variables by name:
+
+```{r}
+mtcars %>% select(cyl, am, vs)
+
+mtcars %>% select(mpg:disp)
+```
+
+For historical reasons, it is also possible to refer an external
+vector of variable names. You get the correct result, but with a note
+informing you that selecting with an external variable is brittle.
```

Comment:
Would it be worth unpacking "brittle" here? (You describe the why in the context, but maybe a parenthetical aside?)

Reply:
This is now:

> You get the correct result, but with a note informing you that selecting with an external variable is ambiguous because it is not clear whether you want a data frame column or an external object.